packages/overreact/src/hooks/helper.js

```diff
@@ -1,10 +1,9 @@
 import _ from 'underscore';

```

Comment:
could we set fetch policy of useRefecth as NetworkOnly in this PR?

Reply:
Why?